src/Files.Filesystem/Helpers/Constants.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace Files.Filesystem.Helpers
+{
+    public static class Constants
```

Comment:
```suggestion
    public static class Constants
```

Reply:
```suggestion
    public static class Constants
```src/Files.Filesystem/Helpers/Constants.cs